src/App.tsx

```diff
@@ -15,36 +16,95 @@
   'Garlic',
 ];
 
+export enum SortType {
+  default = 'default',
+  alphabetically = 'alphabetically',
+  length = 'length',
+}
+
+const getPreperedGoods = (
+  goods: string[],
+  sort: string,
+  isReversed: boolean,
+) => {
+  const preparedGoods: string[] = [...goods];
+
+  switch (sort) {
+    case SortType.alphabetically:
+      preparedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.length:
+      preparedGoods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+};
+
 export const App: React.FC = () => {
+  const [sortBy, setSortBy] = useState(SortType.default);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const visibleGoods = getPreperedGoods(goodsFromServer, sortBy, isReversed);
```

Comment:
Update the function call to `getPreparedGoods` to match the corrected function name.